OSDK.toml

```diff
@@ -27,6 +27,7 @@ args = "$(./tools/qemu_args.sh test)"
 
 [scheme."microvm"]
 boot.method = "qemu-direct"
+build.strip_elf = true
```

Comment:
Why microvm needs to set this value, while other configurations do not?

Reply:
Because if we don't strip it the ELF would be too large. That's an requirement for microvm only (not for `qemu-direct`).

Reply:
Ok. Thanks for the explanation.